### DISCUSSION TITLE
Is a WeekPicker available?

### DISCUSSION THREAD

**Discussion by capoaira** [vuetifyjs/vuetify#18884] [Q&A]
Hey, I'm looking for a WeekPicker, where I can select a week of a Year. I found this closed issue: https://github.com/vuetifyjs/vuetify/issues/4566, but not the feature. Did I miss something?

**Comment by velly** [comment]
TimePicker is not ready yet,

**Reply by capoaira** [reply]
Ok, thank you. Is there a way to get notified about new releases? Then I can check on every new Version if the WeekPicker is ready.

**Comment by inlinecoder** [comment]
Hello guys 👋,

First off, thanks for the awesome set of tool!

A legitimate question: any plans on adding a week selector?
